mappings/net/minecraft/client/realms/gui/screen/RealmsPlayerScreen.mapping

```diff
@@ -30,21 +30,34 @@ CLASS net/minecraft/class_4406 net/minecraft/client/realms/gui/screen/RealmsPlay
 		ARG 1 player
 	METHOD method_21327 drawRemoveIcon (Lnet/minecraft/class_4587;IIII)V
 		ARG 1 matrices
+		ARG 2 x
+		ARG 3 y
+		ARG 4 mouseX
+		ARG 5 mouseY
 	METHOD method_21328 updateOps (Lnet/minecraft/class_4870;)V
 		ARG 1 ops
 	METHOD method_21332 renderMousehoverTooltip (Lnet/minecraft/class_4587;Lnet/minecraft/class_2561;II)V
 		ARG 1 matrices
+		ARG 2 toolTip
```

Comment:
```suggestion
		ARG 2 tooltip
```

Reply:
Gonna grep locally since there is probably more of them